common/i18n/resources/en.js

```diff
@@ -173,6 +173,7 @@ module.exports = {
 
   recordView: {
     justDeleted: 'This record has just been deleted by another user',
+    editTimeout: 'Record editing session has expired',
```

Comment:
It's not only an editTimeout; it can happen also in view mode. I would call this key sessionExpired and the value Record session has expired.

Reply:
Ok